packages/google_sign_in/google_sign_in/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Next
```

Comment:
Since this is an important new feature and it is going to be a minor change. Let's release it with a new version instead of "Next"

Reply:
Done